spec/rails_helper.rb

```diff
@@ -68,6 +68,8 @@
 
   config.before(:each, js: true) do
     allow(Figaro.env).to receive(:domain_name).and_return('127.0.0.1')
+    server = Capybara.current_session.server
+    Rails.application.routes.default_url_options[:host] = "#{server.host}:#{server.port}"
```

Comment:
love this lil patch, I'd been super frustrated by the `example.com` stuff before